app/templates/src/_BoilerplateComponent.js

```diff
@@ -1,17 +1,22 @@
 'use strict';
 <% if (templateLanguage === 'Soy') { %>
-import templates from './<%= componentName %>.soy<% if (buildFormat !== 'amd') { %>.js<% } %>';
+import templates from './<%= componentName %>.soy.js';
 import Component from 'metal-component';
 import Soy from 'metal-soy';<% } else if (templateLanguage === 'JSX') { %>
 import JSXComponent from 'metal-jsx';<% } else { %>
-import Component from 'metal-component';<% } %>
+import Component from 'metal-component';<% } %><% if (buildFormat === 'jquery') { %>
+import JQueryAdapter from 'metal-jquery-adapter';<% } %>
+
+import './<%= kebabCaseName %>.scss';
```

Comment:
I'm not sure if there is a way to deal with `scss` with Webpack in another way. Webpack receives a Javascript entry file and handles everything from that. There is a plugin that we can use to extract the CSS part from the bundle to another file.